selfdrive/car/chrysler/interface.py

```diff
@@ -69,4 +69,4 @@ def apply(self, c):
     if (self.CS.frame == -1):
```

Comment:
Is this even used? CS.frame is never -1

Reply:
ah yeah -1 used to be the default CANParser value until we got CAN data, but now we wait for initialization